src/test/resources/IBMdefined/RecordSeparatedFieldFormat.xsd

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Remove this file due to license. Add to README instructions to copy it into the right spot.